src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier;
+
+    public Lottery() {
+        colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor();
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);
```

Comment:
Avoid using magic numbers, e.g. 101
make them constant field with proper name
```suggestion
        int randomNumber = random.nextInt(101);
```